Readme.md

```diff
@@ -0,0 +1,79 @@
+# SellEverywhere
+
+## Overview
+We want to create a hardware shop using popular technologies in a friendly team
+(MKdir). Examples are any store selling goods. Current idea is to implement an
+admin panel and user panel. The basic interface of the store was designed in [Figma](https://www.figma.com/file/Re4tXW7aNCWOzed7wI1GKe/dirStore).
+
+### Basic flow
+![Basic flow](./docs/basic_flow.png)
+
+### The application must have the following functionality from the administrator perspective:
+
+* authorization - jwt;
+* authentication;
+* creation of a new product (fill price, description, category, category characteristics, name, manufacturer, photo);
+* fill characteristics of the product by chosen category (dynamic characteristics)
+* manage existing products/deliverers (read/edit/delete);
+* promotion ability helps admin to change product price and see history of changes and add product to the list of special propositions
+* add new products/deliverers;
+* view sales statistics for the selected category
+  or popularity of the products for example (diagrams, table), provide analytics;
+* control the processes of users interaction in application;
+* moderation of registered users.
+
+### Admin use case
+![Functionality for ADMIN](./docs/admin_use_case.png)
+
+### The application must have the following functionality from the user:
+
+* authorization - jwt;
+* authentication;
+* browse products by category and its characteristics (dynamic characteristics);
+* adding goods to the cart;
+* view the cart and edit it;
+* provide payment flow for goods;
+* view the history of purchased goods (records of every purchase will be stored with copying detail info);
+* implement the ability to add comments to the specific product (nice to have);
+
+### User use case
+![Functionality for USER](./docs/user_use_case.png)
+
+### Technologies for implementing:
+
+* Frontend: Next.js ssg & ssr, TypeScript, Redux;
+* Database: MySQL / PostgresSQL;
+* Backend: Nest.js, knex;
```

Comment:
connecting Next with Nest wouldn't be to hard for you?

Reply:
@nosovk, we talked together and choose to try to do it because want to learn how to work with this technology. I guess for local development we wouldn`t have any problems. Problems can be with deployment it to versel/heroku, because we do not familiar with it.

Reply:
ok, just in case - you have to start from that step.
https://habr.com/ru/post/570886/
suggests using some tooling https://www.npmjs.com/package/nest-next for integration. You may start from it. I'm not sure that it will work, but it's good starting point (BTW it's much harder then using NextJS without Nest, but you choose)